lightbox.js

```diff
@@ -140,8 +146,17 @@ function bind() {
   });
 }
 
-function init(context) {
-  $context = $(context || document.body);
+function init(options) {
+  var defaults = {
+    context: document.body,
+    bgColor: '#fff',
+    opacity: '0.94'
```

Comment:
if you are setting bgColor and opacity defaults here, should they be removed from the scss?